src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
Thanks for pointing out.

>So in this case you should use Utils::String::toStdString().

Doing this way there is one extra conversion to `std::string` before to `char *`. So I choose `program.toUtf8().constData()`.

>Btw, did you test with utf8 + windows + non-latin path? Did it work?

It works.